.github/workflows/pnl-ci-docs.yml

```diff
@@ -43,16 +43,19 @@ jobs:
       uses: actions/checkout@v2.3.5
       if: ${{ matrix.pnl-version == 'head' }}
       with:
-        fetch-depth: 10
+        fetch-depth: 200
         ref: ${{ github.ref }}
 
     - name: Checkout pull base
       uses: actions/checkout@v2.3.5
       if: ${{ matrix.pnl-version == 'base' }}
       with:
-        fetch-depth: 10
+        fetch-depth: 200
         ref: ${{ github.base_ref }}
 
+    - name: Checkout tags
+      run: git fetch --tags
```

Comment:
I'm not sure I understand the goal here.
The main docs page is built on tag events so it already has the correct tag/version,
the branch docs are currently identified with as e.g. "0+untagged.11.gaabca55".
Is the idea here to to have instead something like "v0.9.1.1-67-gaabca55987" ?
the identifying part (short git hash) is the same.

Reply:
Yes exactly, modeci_mdf depends on a specific psyneulink version, and if the local version of psyneulink is untagged, the installation will fail ([example](https://github.com/kmantel/PsyNeuLink/runs/3902174562?check_suite_focus=true)). I think this is due to the more restrictive dependency checks in pip 20.3. 

If the repo doesn't have older tags and a history from the current commit to one of those tags (reason for fetch-depth change above), versioneer will leave the version as untagged[...]

Reply:
ah, that explains it, thanks! Can you add a short comment around tag fetching step?
so modeci_mdf has circular dependency on pnl?

Reply:
Sure, I will, and yes modeci_mdf depends on psyneulink as well